nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -197,7 +196,10 @@ let
     ]}
 
     if [ "\$textmode" == "false" ]; then
-      terminal_output gfxterm
+      if loadfont (\$root)/EFI/BOOT/grub-theme/dejavu.pf2; then
```

Comment:
This doesn't seem right to me. Shouldn't the condition be in a logical-and with the `if` statement above it, rather than  in a nested `if` statement?

Reply:
You said on Discourse that you tried `unicode.pf2` and that would always pass. Do different fonts pass for different reasons? Why is *this font* the one that best illustrates that we should use `gfxterm`?